### ISSUE TITLE
Format for course contents

### ISSUE THREAD

**Issue by girlscript** [girlscript/contribute-girlscript#4]
The course contents should be technical, easy-to-learn and original work of the contributor, The content should be in the form of a presentation(PPt) or must be a PDF. We are currently **not planning** to have video lectures on the website.
Please refer to the sample here-
![ss2](https://cloud.githubusercontent.com/assets/27850350/26230048/b1cc7876-3c63-11e7-908d-ec398df49733.PNG)

**Comment by jgmac1106** [comment]
Do you need to create all new content or can you just curate existing materials? 

I get the low bandwidth. Smart idea. Are you thinking English or Hindi?

**Comment by girlscript** [comment]
We can curate the existing materials, if the provider of the content don't have any issues. We will give credits on the website for contribution.
We want our contents in English.

**Comment by jgmac1106** [comment]
There is plenty of open source material and all of the MDN content. If you want to schedule a chat next week we can. I was looking for a global sprint proje t to contribute to and your's sounds perfect.

**Comment by girlscript** [comment]
Sure whenever you are free, we can chat regarding how we can move further. We can chat here- https://gitter.im/Girl-script/Contents

**Comment by siddhartharao17** [comment]
I'm still trying to understand the submission of the content. Do you mean we create a PDF of all the content that we wanna contribute and then submit here on the thread?

**Comment by girlscript** [comment]
Yes we have to create pdf in a summarized form and in easy language. People can read the PDF or download it. For now, we don't want to make videos, first want to gather written contents.
